src/protocols/index.js

```diff
@@ -34,6 +37,7 @@ module.exports = class Protocol {
 
   static availableNetworks() {
     return immutable.fromJS({
+      arweave: ['mainnet'],
```

Comment:
The way `graph-node` currently is, I think you'll have to add a _prefix_ here, just like we've done for NEAR (`'near-mainnet'`).

Can you confirm this? @leoyvens or @maoueh

Reply:
`arweave-mainnet` now